minos/networks/discovery/clients/kong.py

```diff
@@ -0,0 +1,33 @@
+from .abc import (
+    DiscoveryClient,
+)
+
+
+class KongDiscovery(DiscoveryClient):
```

Comment:
I think it's preferable to name it as `KongDiscoveryClient` to follow the same naming convention for all the `DiscoveryClient` heirs.

Reply:
Done.